.github/workflows/scripts/linux/docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+cd /pcsx2
+
+mkdir build && cd build || exit 1
```

Comment:
This might be redundant since `-e` is set.
```suggestion
mkdir build && cd build
```